src/main/java/net/mcreator/ui/workspace/ModElementCodeDropdown.java

```diff
@@ -68,12 +68,15 @@ class ModElementCodeDropdown extends JPopupMenu {
 					listMenu.setIconTextGap(8);
 					listMenu.setBorder(BorderFactory.createEmptyBorder(10, 0, 11, 0));
 
+					AtomicInteger files = new AtomicInteger(0);
 					fileList.forEachTemplate(listTemplate -> {
-						if (listTemplate.getFile().isFile())
+						if (listTemplate.getFile().isFile()) {

```

Comment:
Is isFile check even needed here, this will only run if condition passes, so it will be a file at this point?

Reply:
When I was testing iteration refactoring, I tried adding conditions to list templates section of YAML file for tags and on the next launch this dropdown gave me files that were not generated previously, I thought this was a bug and fixed it with this check, but I forgot that's not a thing an end user would do xD

Reply:
✔️ Fixed